homeassistant/components/zwave_js/config_flow.py

```diff
@@ -309,6 +332,44 @@ async def async_step_user(
 
         return await self.async_step_manual()
 
+    async def async_step_usb(self, discovery_info: dict[str, str]) -> FlowResult:
+        """Handle USB Discovery."""
+        if not is_hassio(self.hass):
+            return self.async_abort(reason="discovery_requires_supervisor")
+
+        vid = discovery_info["vid"]
+        pid = discovery_info["pid"]
+        serial_number = discovery_info["serial_number"]
+        device = discovery_info["device"]
+        manufacturer = discovery_info["manufacturer"]
+        description = discovery_info["description"]
+        # The Nortek sticks are a special case since they
+        # have a Z-Wave and a Zigbee radio. We need to reject
+        # the Zigbee radio.
+        if (
+            vid == "10C4"
+            and pid == "8A2A"
+            and "Z-Wave" not in discovery_info["description"]
+        ):
+            return self.async_abort(reason="not_zwave_device")
+
+        await self.async_set_unique_id(
+            f"{vid}:{pid}_{serial_number}_{manufacturer}_{description}"
+        )
+        self._abort_if_unique_id_configured()
+        dev_path = await self.hass.async_add_executor_job(get_serial_by_id, device)
+        self.usb_path = dev_path
+        self._title = _format_port_human_readable(
+            dev_path,
+            serial_number,
+            manufacturer,
+            description,
+            vid,
+            pid,
+        )
+        self.context["title_placeholders"] = {CONF_NAME: self._title}
+        return await self.async_step_on_supervisor({CONF_USE_ADDON: True})
```

Comment:
There needs to be a confirmation form before we move on with the flow and start setting up the add-on etc. See the hassio steps.

Reply:
Will do